### DISCUSSION TITLE
High virtual memory usage

### DISCUSSION THREAD

**Discussion by actcwlf** [apache/doris#28535] [A - General / Q&A]
I found that the virtual memory usage is very high, is this a normal status?

![image](https://github.com/apache/doris/assets/32570929/3d1fa963-4158-495b-ac3a-ada6c8e107df)

**Comment by LemonLiTree** [comment]
which version？

**Reply by actcwlf** [reply]
2.0.3

**Reply by actcwlf** [reply]
Is there any update?
